### ISSUE TITLE
Allow converting every filetype to markdown by default

### ISSUE THREAD

**Issue by benlubas** [GCBallesteros/jupytext.nvim#7]
I'd pretty much like to be able to set these options all at the top level and have them work the way they currently do when they're in the `custom_lanugage_formatting.python` table, just for all languages.

```lua
{
    extension = "md",
    style = "markdown",
    force_ft = "markdown",
}
```

I'm currently not aware of a way to do this

**Comment by GCBallesteros** [comment]
If I understand correctly you want to be able to configure the `extension` and `force_ft` options in the same way you can do with the style right now. That's perfectly reasonable. I will make it happen next week.

**Comment by GCBallesteros** [comment]
Closed by #11
